client/state/data-layer/wpcom/me/account/close/index.js

```diff
@@ -43,7 +43,19 @@ export function receiveAccountCloseSuccess() {
 	return closeAccountSuccess();
 }
 
-export function receiveAccountCloseError() {
+export function receiveAccountCloseError( action, error ) {
+	if ( error.error === 'active-subscriptions' ) {
+		return errorNotice(
+			translate( 'This user account cannot be closed while it has active subscriptions.' )
+		);
+	}
+
+	if ( error.error === 'active-memberships' ) {
+		return errorNotice(
+			translate( 'This user account cannot be closed while it has active memberships.' )
```

Comment:
As discussed internally, I'd probably go with something like "purchases" for this, because "memberships" is (mostly) a code-level name that isn't exposed to users.

The official name of the feature is Payments, I guess, but that is pretty generic, and one reason I was thinking "purchases" is that the user will be able to view and cancel these on the Purchases screen at https://wordpress.com/me/purchases.

Reply:
Something like "purchases" paired with an adjective seems like it my clarify what's going on. For example, "active purchases" would imply there is something about the purchase that is considered ongoing, which would be reflected in the "Never Expires" seen on the Purchases screen.

It's worth noting that the language used to disable these one-time payments is pretty awkward:

![image](https://user-images.githubusercontent.com/38750/116281588-79a45580-a74f-11eb-8ffa-143bc5508105.png)

Reply:
It already has that adjective, so 👍